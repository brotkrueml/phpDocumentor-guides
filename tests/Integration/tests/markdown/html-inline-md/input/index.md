# Markdown with html

In files of text, where words take flight,<br/>
Markdown weaves its magic, bold and bright.<br>
Hashes and stars, a simple code,<br >
A poet's playground, where stories unfold. <BR>
