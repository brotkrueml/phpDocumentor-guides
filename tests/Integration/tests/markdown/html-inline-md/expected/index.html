<!DOCTYPE html>
<html lang="en">
<head>
    <title>Markdown with html - Project Title</title>

</head>
<body>
    <h1>Markdown with html</h1>

<p>In files of text, where words take flight,Markdown weaves its magic, bold and bright.Hashes and stars, a simple code,A poet&#039;s playground, where stories unfold. &lt;BR&gt;</p>
</body>
</html>
